### ISSUE TITLE
Add help how to open app drawer in Support Q&A

### ISSUE THREAD

**Issue by georg-d** [Deletescape-Media/Lawnchair#2657]
### Describe the bug

When Lawnchair is the start app, opning the app drawer works a little different than with default quickstep launcher. Currently, it's difficult to find the answer on the public web – so please add it in Lawnchair Support https://lawnchair.freshdesk.com/support/solutions 

### Steps to reproduce

1. with default launcher quickstep, swipe up not at lower border of the screen
2. observe app drawer opens
3. install latest lawnchair from Google Play (currently v2.0-2589) and set as default launcher
4. swipe up not at lower border of the screen
5. observe switching apps openes

### Expected behavior

In [Lawnchair Support](https://lawnchair.freshdesk.com/support/solutions), add an entry like

Q: Swiping up from screen edge does not open app-drawer. How to open it?

A: Start swipe up not _at lower border_ of the screen, but instead anywhere _within_ the screen (i.e. above the virtual back & home buttons). Then you will see the app drawer listing all installed apps.

### Device information

- Device: Fairphone 3 
- OS: stock ROM Android version 10.0
- App version 2.0-2589

### Additional context

Maybe this is/was the solution for https://github.com/LawnchairLauncher/lawnchair/issues/1792

**Comment by SuperDragonXD** [comment]
This issue may be closed because it doesn’t relate to Lawnchair 12.1 (latest version). Unfortunately, the devs don't support the older Lawnchair versions anymore.

(Also, that help desk is outdated too.)

**Comment by georg-d** [comment]
If there is an 12.1 _release_ that is the only supported one, it shall better be pushed to google play store 😉 I found v12.x at https://github.com/LawnchairLauncher/lawnchair/releases but all v12.x are alpha sate – which is for obvious reasons not suited for the avagerage user. The combination makes it difficult to understand that all issues concerning older releases are plain simply rejected, even if it's about documentation that is very likely also applying to v12 😕

If the help desk at freshdesk.com is outdated, it would be user friendly to note that there and to set a link set to the current help desk. At least I did not find one within 15min research. Maybe it is hidden behind login walls of telegram, twitter or the like.
